### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by Habtamu210**
### In this project: 
- created a full website for the Math magicians app
- Create a website consisting of 3 pages: Home, Calculator, and Quote

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
   return (
-    <div className="App">
-      <div className="background" />
-      <Calculator />
-    </div>
+    <BrowserRouter>
+      <Navigation />
+      <Routes>
+        <Route element={<div className="back-page" />} />
+        <Route path="/mathmatica" element={<Home />} />
```
Comment (levy002) [inline src/App.js]: - [ ] KIndly make sure when the application is started goes directly to the `Home` page. In order to achieve that you have to make the `Home` page the index page so whenever we are on `path="/"` this page is rendered directly.

   _**Hint: `<Route index element={<Home />} />`**_

File: src/components/navagation.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  <nav className="navigation">
+    <div className="logos">
+      <Link to="/">
+        Mathmatica
+      </Link>
+    </div>
+    <div className="menu">
+      <li>
+        <Link to="/mathmatica">Home</Link>
+      </li>
```
Comment (levy002) [inline src/components/navagation.jsx]: - [ ] As mentioned above, make sure that this `Home` page is the `index` page, in order to achieve that you have to change where this link navigates.

  _**Hint: `<Link to="/">Home</Link>`**_

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Habtamu210   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good UI :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to remove all comments in the README in order to keep your repo clean and professional.

  
   image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/navagation.jsx (hunk 2 above)
Comment (rotimiazeez) [reply src/components/navagation.jsx]: As suggested by previous reviewer, your `Home` page should be the index page of your APP. Kindly ensure the link looks something like `Hint: <Link to="/">Home</Link>` so it re-direct you to the index page when you click on home 🏡👌🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
Hi @Habtamu210  😉, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
## Highlights ✔️
- Good UI ✔️
- No linter errors ✔️
- Good PR description 👌🏽 


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: src/components/navagation.jsx (hunk 2 above)
Comment (rotimiazeez) [reply src/components/navagation.jsx]: Good job implementing this suggestion 😉

**Review by DammyShittu — APPROVED** [review]
Hi @Habtamu210,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**